python_modules/dagster/dagster/_core/definitions/automation_condition_sensor_definition.py

```diff
@@ -79,6 +81,8 @@ class AutomationConditionSensorDefinition(SensorDefinition):
             sensor. The actual interval will be longer if the sensor evaluation takes longer than
             the provided interval.
         description (Optional[str]): A human-readable description of the sensor.
+        allow_backfills (bool): If set to True, will emit a backfill on any tick where more than one partition
+            of any single asset is requested, rather than individual runs. Defaults to False.
```

Comment:
I find the naming weird. `allow_backfils` implies that the user has the _option_ to allow backfills. That is not really true. The user is _instructing_ the system to emit backfills, whenever possible right?

`emit_backfills`? This has the downside I guess of implying that it always emits backfills, even with single partition.

* `merge_runs`
* `emit_only_runs`

Other ideas?